### ISSUE TITLE
Working Group Call #6

### ISSUE THREAD

**Issue by lrettig** [stacksgov/pm#19]
Date/time: Wednesday, April 1, 2020, 7am PT/10am ET/2pm UTC/4pm CEST/10pm CST ([time zone conversion](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Stacks+Governance+Working+Group+Call&iso=20200401T10&p1=179&ah=1))
**Note recent timezone changes. Please double-check the time of this call in your locale using the above link!**
Location: Link will be posted in [#governance working group channel](https://discordapp.com/invite/ny6wGkx) on Discord shortly before the call
Recording: TBA
Moderator: @joberding 

If you'd like to be added to the recurring calendar invitation, please DM me your email address on Discord.

Proposed agenda (feel free to amend!):
- What happened over the past week? Standup!
  - What is everyone working on right now?
  - Any blockers?
  - What will happen over the next week?
- Community survey
- One-on-one interviews and notes
- Preliminary governance proposal
- Stacks Foundation update (from PBC)
- Any other updates?
- Next/recurring call schedule
- Time allowing: Harold to discuss indigenous nations and matrilineal governance structures

**Comment by lrettig** [comment]
@whoabuddy @joberding would one of you like to moderate next week's call?

**Comment by whoabuddy** [comment]
@joberding would you like to moderate and I will work on [writing update 4](https://github.com/stacksgov/updates/issues/7) in updates?

**Comment by joberding** [comment]
Confirming that I accepted moderation for this week.

**Comment by lrettig** [comment]
My update for today's call:
- Reviewing Brittany's governance research notes
- Survey: As of right now we have 32 responses (that's +24 from when we released it). We're getting there! But lots more work to do advertising this and recruiting participants. The results land in a Google sheet, but it contains some sensitive info such as email addresses. Let's figure out the right way (and timing) to share this.
- Governance proposal: I migrated a lot of the content of the proposal to a new GitHub repo: https://github.com/stacksgov/proposals/issues. It's open for comment. Feel free to add your own proposals. (For now, it doesn't contain specific proposals for Foundation governance because this hasn't been determined yet.)
- Interviews: I've begun to drop the notes here: stacksgov/resources#23.
- Next tasks: Writing down [PM best practices](https://github.com/stacksgov/pm/issues/20) and [SIP governance proposal](https://github.com/stacksgov/proposals/issues/1).
- Contributors: Thank you to Julia for offering to moderate this call, and to Jason for offering to take notes and write the next weekly update! They've both been added as owners of the GitHub org. and the YouTube channel.
